codec_specs.md

```diff
@@ -190,7 +190,7 @@ Codec Name: AVC/H.264
 
 Description: Individual pictures of AVC/H.264 stored as described in [@!ISO.14496-15.2014].
 
-Initialization: The `Private Data` contains a `AVCDecoderConfigurationRecord` structure as defined in [@!ISO.14496-15.2014].
+Initialization: The `Private Data` contains a `AVCDecoderConfigurationRecord` structure as defined in [@!ISO.14496-15.2014]; for legacy reasons ([Block Addition Mappings](#block-addition-mappings) are prefered), the `AVCDecoderConfigurationRecord` structure MAY be followed by an extension block beginning with a 4-byte extension block size field which is the size of the extension block minus 4 (excluding the size of the extension block size field) and a 4-byte field corresponding to `BlockAddIDType` followed by a content corresponding to the content of `BlockAddIDExtraData`.
```

Comment:
I forgot to mention it in the other PR, but as this exception for legacy files is about one specific use case (`mvcC`) maybe we should limit the exception to that one possible type.